qiita_pet/webserver.py

```diff
@@ -93,9 +95,16 @@ def __init__(self):
             (r"/stats/", StatsHandler),
             (r"/download/(.*)", DownloadHandler),
             (r"/vamps/(.*)", VAMPSHandler),
-            # 404 PAGE MUST BE LAST IN THIS LIST!
-            (r".*", NoPageHandler)
         ]
+        if qiita_config.portal == "QIITA":
+            # Add portals editing pages only on main portal
+            portals = [
+                (r"/admin/portals/studies/", StudyPortalHandler)
+            ]
+            handlers.extend(portals)
```

Comment:
Why are you using extend? You can use a simple append:
handlers.append((r"/admin/portals/studies/", StudyPortalHandler))